flutter_google_places_sdk_ios/ios/Classes/SwiftFlutterGooglePlacesSdkIosPlugin.swift

```diff
@@ -354,8 +355,17 @@ public class SwiftFlutterGooglePlacesSdkIosPlugin: NSObject, FlutterPlugin {
         return CLLocation(latitude: lat, longitude: lng)
     }
     
-    private func initialize(apiKey: String?) {
+    private func initialize(apiKey: String?, locale: Dictionary<String,Any>?) {
+        defineLocale(locale: locale)
         GMSPlacesClient.provideAPIKey(apiKey ?? "")
         placesClient = GMSPlacesClient.shared()
     }
+
+    private func defineLocale(locale: Dictionary<String,Any>?) {
+        if let locale = locale,
+           let language = locale["language"],
+           let country = locale["country"] {
+            UserDefaults.standard.set(["\(language)-\(country)"], forKey: "AppleLanguages")
```

Comment:
@bruno94silva if the changes we are doing are related to setting this UserDefaults value for key AppleLanguages, why don't we do this outside the plugin?